src/node/entry.ts

```diff
@@ -79,14 +79,20 @@ const main = async (args: Args): Promise<void> => {
   const serverAddress = await httpServer.listen()
   logger.info(`HTTP server listening on ${serverAddress}`)
 
-  if (auth === AuthType.Password && !process.env.PASSWORD) {
-    logger.info(`  - Password is ${originalPassword}`)
-    logger.info("    - To use your own password set the PASSWORD environment variable")
-    if (!args.auth) {
-      logger.info("    - To disable use `--auth none`")
+  if (!args.auth) {
```

Comment:
I think we have an edge case with the auth. For example my output says it's using the password from the configuration file but I don't have a password in there, just `bind-addr`. I don't have `auth` in there either. I can access code-server without a password.

I think we might have to always write the generated password if it's missing to the config file to fix this? Or if there's a config file without a password then we generate a new one but don't write it (so in this case it would be different each time again like before).

Reply:
Yea we'll have to always write a generated pass to the config file.